packages/api/tests/rich-text-detection.test.ts

```diff
@@ -319,9 +325,12 @@ describe('detectFacets', () => {
           },
         ],
       ],
+      ['this #​ should not be a tag', [], []],
+      ['this #​a should not be a tag', [], []],
+      ['this #a​b should be a tag', ['a'], [{ byteStart: 5, byteEnd: 7 }]],
```

Comment:
Same question, I'm guessing there's a zero-width char in there?

Reply:
Same same